sql/changes/1.8/reconciliation-links-cascade.sql@1

```diff
@@ -0,0 +1,10 @@
+
+
+alter table cr_report_line_links
+   drop constraint cr_report_line_links_report_line_id_fkey,
+   add CONSTRAINT cr_report_line_links_report_line_id_fkey
+        FOREIGN KEY (report_line_id)
+           REFERENCES public.cr_report_line (id) MATCH SIMPLE
```

Comment:
We're adding back a fixed reference to public?

Reply:
This is a copy of the original file for the purpose of the SHA calculation: that is used to prevent it from being applied again. The `@1` part means that it's an older version of the file that should be applied if none of the SHA sums match.

The file without the public prefix in it, is the one that's going to be run on the database going forward.